vite.config.ts

```diff
@@ -114,19 +115,92 @@ export default defineConfig(({ command: _buildOrServe }) => ({
 			},
 		},
 		{
-			name: 'update-animations-json',
-			handleHotUpdate({ file, server, read: _read }) {
-				if (file.endsWith('animations.json')) {
+			name: 'create-dist-files',
+			apply: 'serve',
+			buildStart() {
+				const files = [...moduleJSON.esmodules, moduleJSON.styles].flat()
+				for (const name of files) {
+					fs.writeFileSync(`${name}`, '', { flag: 'a' })
+				}
+			},
+		},
+		mergeAnimationsPlugin(),
+	],
+}))
+
+function mergeAnimationsPlugin(): PluginOption {
+	const mergeAnimations = () => {
+		let animations = {}
+		const keys = new Set<string>()
```

Comment:
Is this necessary? Why not compare the newly added keys directly to those inside the `animations` object?